docker-compose/keycloak.yaml

```
@@ -0,0 +1,39 @@
+services:
+  keycloak:
+    image: quay.io/keycloak/keycloak:22.0.5
+    command:
+      - "start-dev"
+    environment:
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+      KC_HEALTH_ENABLED: true
+      KC_HOSTNAME_STRICT_BACKCHANNEL: false
+      KC_HOSTNAME: localhost
+    ports:
+      - 8080:8080
```

isn't that port almost always in use?

```suggestion
      - 1852:8080
```